modules/logger/parser/parser.go

```diff
@@ -140,6 +140,7 @@ func parseAndStoreTestOutput(
 
 			case isStatusLine(data):
 				testName := getTestNameFromStatusLine(data)
+				previousTestName = testName
```

Comment:
This is the fix for the lost error trace. The error trace line was happening right after a `=== CONT TESTNAME` line, which is a status line. Now we roll up any unmarked test lines that show up after a status line to the test that relates to the status line.